README.md

```diff
@@ -1,86 +1,99 @@
-# OpenApocalypse
- 
+# OpenApocalypse [![Linux Build Status](https://travis-ci.org/JonnyH/OpenApoc.svg?branch=master)](https://travis-ci.org/JonnyH/OpenApoc) [![Windows Build Status](https://ci.appveyor.com/api/projects/status/07ndsvrbyct924a1)](https://ci.appveyor.com/project/JonnyH/openapoc)
```

Comment:
This reminds me just how silly it is to have the 'upstream' github on one user (pmprog), and all the CI stuff on another (mine) - so they'll never quite be in sync.

Hopefully I'll be changing that this weekend, by moving everything to an organisation, so we can have multiple 'owners' instead of putting all the burden on @pmprog - but that requires him to transfer ownership first.

Once that's done, we can update the links and merge.